Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -2435,6 +2459,47 @@ void webkit_settings_set_pictograph_font_family(WebKitSettings* settings, const
     g_object_notify_by_pspec(G_OBJECT(settings), sObjProperties[PROP_PICTOGRAPH_FONT_FAMILY]);
 }
 
+/**
+ * webkit_settings_get_math_font_family:
+ * @settings: a #WebKitSettings
+ *
+ * Gets the #WebKitSettings:math-font-family property.
+ *
+ * Returns: (nullable): The default font family used to display content marked with math font.
+ *
+ * Since 2.52
+ */
+const gchar* webkit_settings_get_math_font_family(WebKitSettings* settings)
+{
+    g_return_val_if_fail(WEBKIT_IS_SETTINGS(settings), 0);
+
+    return settings->priv->mathFontFamily.data();
```

Comment:
I fear this might return an empty string rather than nullptr? We'll find out from the API test.